lte/gateway/Vagrantfile

```diff
@@ -195,4 +195,37 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   end
 
+  config.vm.define :magma_deb, autostart: false do |magma_deb|
+
+    magma_deb.vm.box = "ubuntu/focal64"
+    magma_deb.vm.box_version = "20220804.0.0"
+    magma_deb.vm.hostname = "magma-deb"
+    magma_deb.vbguest.auto_update = false
+
+    # Create a private network, which allows host-only access to the machine
+    # using a specific IP.
+    magma_deb.vm.network "private_network", ip: "192.168.60.142", nic_type: "82540EM"
```

Comment:
This IP config is same as the `magma` VM. Could you please add a note here that only of these VMs should be running at a time and any updates to `magma` VM networking should be added here too.

Reply:
Thanks for the suggestion! I've just added these.